app/views/carriers/_filter_form.html.erb

```diff
@@ -59,6 +59,7 @@
   <div class="row">
     <div class="col">
       <%= link_to "Reset Filters", reset_filterrific_url, class: 'btn btn-secondary btn-sm' %>
+      <label><%= f.check_box(:with_overdue) %><%= "Overdue" %></label>
```

Comment:
@sarslanoglu As I mentioned, this functionality should go on the loan listing (`LoanListingController`), not the carriers index. It is the loan that is overdue. In some cases it might make sense to put it here (or even in both places), but we have had discussions with the client that we should avoid having multiple ways to do some things because it can cause confusion.

Since listing the overdue loans can already be done on the loans listing (by setting the filters manually as I mentioned in my original comment), this just becomes a matter of adding a new button that sets the filters to this state automatically (probably just in jquery, but let me know if you have issues with that or if you have another suggestion). In that case no additional scope(s) would be needed (on the carriers or the loans).